at-tanvo/D3-Exercise5.rb

```diff
@@ -0,0 +1,10 @@
+puts "Nhap phan tu mang :"
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/atruby-team/internship-18-summer/blob/7587b87b7304d151b3838535fcf67736f443399f/at-tanvo/D3-Exercise5.rb#L1"></span>Style/StringLiterals: Prefer single-quoted strings when you don’t need string interpolation or special symbols. (https://github.com/rubocop-hq/ruby-style-guide#consistent-string-literals)</td>
    </tr>
  </tbody>
</table>